projects/gnu.org/findutils/package.yml

```diff
@@ -0,0 +1,50 @@
+distributable:
+  url: https://ftp.gnu.org/gnu/findutils/findutils-{{version}}.tar.xz
+  strip-components: 1
+
+versions:
+  url: https://ftp.gnu.org/gnu/findutils/
+  match: /findutils-\d+\.\d+\.\d+\.tar\.xz/
+  strip:
+    - /^findutils-/
+    - /\.tar\.xz$/
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+
+  script:
+    - ./configure $ARGS
+    - make --jobs {{ hw.concurrency }} install
+
+  env:
+    ARGS:
+      - --prefix="{{prefix}}"
+      - --localstatedir={{prefix}}/var/locate
+      - --disable-dependency-tracking
+      - --disable-nls
+      - --with-packager=Tea.xyz
+      - --with-packager-bug-reports=https://github.com/teaxyz/pantry/issues
+    darwin:
+      ARGS:
+        - --program-prefix=g
+
+provides:
+  darwin:
+    - bin/gfind
+    - bin/glocate
+    - bin/gupdatedb
+    - bin/gxargs
```

Comment:
I'd skip the prefixing on darwin; we have omitted this for other gnu tools, since tea allows controlling what you get.

Reply:
> I'd skip the prefixing on darwin; we have omitted this for other gnu tools, since tea allows controlling what you get.

Oh, okay. I will change :)